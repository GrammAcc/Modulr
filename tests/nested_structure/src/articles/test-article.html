<!DOCTYPE html>
<html lang="en">
  <head>
    <title>Test Article</title>
  </head>
  <body>
    <!-- modulr-component :nav: -->
    <main>
      <div>Test Article Body</div>
    </main>
  </body>
</html>
