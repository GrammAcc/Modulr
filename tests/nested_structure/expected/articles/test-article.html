<!DOCTYPE html>
<html lang="en">
  <head>
    <title>Test Article</title>
  </head>
  <body>
    <nav>
      <button>Home</button>
      <button>About</button>
    </nav>
    <main>
      <div>Test Article Body</div>
    </main>
  </body>
</html>
